doc/nrf/samples/matter.rst

```diff
@@ -18,48 +18,56 @@ The following table lists variants and extensions available out of the box for e
       - Door lock
       - Template
       - Window covering
+      - Thermostat
     * - FEM support
       - ✔
       - ✔
       - ✔
       - ✔
       - ✔
+      -
```

Comment:
@markaj-nordic , can you help me fill out this table for thermostat?

Reply:
done

Reply:
Thanks